website/src/utils/layer-params.js

```diff
@@ -56,7 +56,11 @@ export function getLayerParams(layer, propParameters = {}) {
     if (propParameters[key]) {
       paramsArray.push({name: key, ...propParameters[key]});
     } else {
-      const param = propToParam(key, layer.constructor._propTypes[key], layer.props[key]);
+      const LAYER_PROPTYPES = layer.constructor._propTypes[key];
+      if (layer.constructor.name === 'ColumnLayer') {
```

Comment:
I'm not a fan of special cases like this. Can we scale up the `getElevation` prop instead and keep the default range?

Reply:
Yes, that seems like a more obvious option since we have access to all the props of `ColumnLayer`. Will update the code.